src/apis/setMaxSignaturesCount.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Set the number of signatures that can be store in the WebViewer (default is 2)
```

Comment:
Should this be `stored`?

Reply:
You're correct. I need to add a grammar check plugin for VS code when I have time